benchmark_test.go

```diff
@@ -33,14 +33,15 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/sirupsen/logrus"
 	"go.uber.org/zap"
+	"k8s.io/klog/v2/klogr"
 )
 
 func init() {
 	os.Stderr, _ = os.Open("/dev/null")
 
 	// stdr
 	stdr.SetVerbosity(1)
-	// globr
+	// glogr / klogr
```

Comment:
This doesn't actually bump verbosity for klog because `klog.InitFlags` wasn't called. It cannot be called because the glog and klog would conflict because both would try to register the same flag.

I don't think it matters because we log at V(0) (enabled by default) and V(9) (disabled by default).

Does it matter for glog? If not, then this whole `flag.Set("v", "1")` should get removed.

Reply:
I haven't use `glog` much.